quartz/components/styles/search.scss

```diff
@@ -86,93 +86,133 @@
         }
       }
 
-      & > #results-container {
-        & .result-card {
-          padding: 1em;
-          cursor: pointer;
-          transition: background 0.2s ease;
-          border: 1px solid var(--lightgray);
-          border-bottom: none;
-          width: 100%;
-          display: block;
-          box-sizing: border-box;
-
-          // normalize card props
-          font-family: inherit;
-          font-size: 100%;
-          line-height: 1.15;
-          margin: 0;
-          text-transform: none;
-          text-align: left;
-          background: var(--light);
-          outline: none;
-          font-weight: inherit;
-
-          & .highlight {
-            color: var(--secondary);
-            font-weight: 700;
-          }
+      & > #search-layout {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
 
-          &:hover,
-          &:focus {
-            background: var(--lightgray);
-          }
+        & > * {
+          height: calc(75vh - 20em);
+          background: none;
+          border-radius: 5px;
+          border: 1px solid var(--lightgray); // Border to define the box
+        }
 
-          &:first-of-type {
-            border-top-left-radius: 5px;
-            border-top-right-radius: 5px;
+        @media all and (max-width: $mobileBreakpoint) {
+          display: block;
+          & > * {
+            border: none !important;
           }
 
-          &:last-of-type {
-            border-bottom-left-radius: 5px;
-            border-bottom-right-radius: 5px;
-            border-bottom: 1px solid var(--lightgray);
+          & > *:not(#results-container) {
+            display: none !important;
           }
```

Comment:
is there any way to do this without the `!important` selector?

Reply:
oh this is so that the preview-container doesn't show on mobile. I couldn't figure out a way not to use !important, let me test a bit and I will get back to you.

Reply:
<img width="435" alt="Screenshot 2024-01-31 at 02 49 36" src="https://github.com/jackyzha0/quartz/assets/29749331/8df81a8a-25b3-4725-b0f8-daa96a27fc9a">

Hmm this is what on phone would look like, without display: none !important

I think this looks fine on phone?

Reply:
how do you trigger the preview on mobile? just tap? and two taps to get to the page?

Reply:
Just tap

Reply:
so on mobile it only ever previews the first item? could we maybe just default to old layout on mobile

Reply:
yep sounds good.